clients/unshaded/src/main/java/tachyon/client/TachyonFS.java

```diff
@@ -31,14 +31,17 @@
 import tachyon.Constants;
 import tachyon.TachyonURI;
 import tachyon.annotation.PublicApi;
+import tachyon.client.block.BlockMasterClient;
```

Comment:
yes, the clients has been moved from tachyon.client into tachyon.client.{block,file,table}